platform/golden/bullseye.pkr.hcl

```diff
@@ -93,22 +77,36 @@ source "virtualbox-iso" "base-debian-amd64" {
   iso_checksum         = "${var.iso_checksum_type}:${var.iso_checksum}"
   iso_url              = "${var.iso_url}"
   iso_interface        = "virtio"
-  memory               = "${var.memory}"
   nic_type             = "virtio"
   hard_drive_interface = "virtio"
   rtc_time_base        = "UTC"
-  shutdown_command     = "echo '${var.ssh_password}' | sudo -S shutdown -P now"
-  ssh_username         = "${var.ssh_username}"
+  shutdown_command     = "sudo -S shutdown -P now"
+  ssh_username         = "packer"
   ssh_password         = "${var.ssh_password}"
   ssh_wait_timeout     = "30m"
   vboxmanage = [
+    # enable recording video of install process, for debug and build record
     [ "modifyvm", "{{.Name}}", "--recording", "on" ],
-    [ "modifyvm", "{{.Name}}", "--nat-localhostreachable1", "on" ]
+    # enable proxy access across NAT network interface.
+    [ "modifyvm", "{{.Name}}", "--nat-localhostreachable1", "on" ],
```

Comment:
No need to close this comment, this is for others that may look at these PRs later, if ever.

This is a change in default behavior introduced in a recent virtual box update:
https://github.com/hashicorp/packer-plugin-virtualbox/issues/104